static/src/javascripts/__flow__/types/acquisitions.js

```diff
@@ -12,6 +12,7 @@ declare type AcquisitionsEpicTemplateCopy = {
 };
 
 declare type EngagementBannerTemplateParams = {
+    leadSentence: string,
```

Comment:
should this field be optional? Not all banner templates will use it

Reply:
it will just get empty string, but maybe empty is better

Reply:
done that! thanks